Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -23,23 +23,7 @@ def linkage
         result.display_test_output
         Homebrew.failed = true if result.broken_dylibs?
         if OS.linux?
```

Comment:
I don't feel strongly about either in this case. I usually a hypothetical third operating system supported by `brew` as my litmus test for which it should be. In this case I'm not sure what the dynamic library system of the hypothetical third operating system would be.